qds_sdk/cluster.py

```diff
@@ -66,9 +66,10 @@ def list(cls, state=None):
         Returns:
             List of clusters satisfying the given criteria
         """
-        conn = Qubole.agent()
+        conn = Qubole.agent(version=Cluster.api_version)
```

Comment:
If we change the default version to v1.3 at all places, then we would not need this change. 
Also I think we pass the version info to various fns of class `Cluster` via qds.py. For example check `update()`, `clone()`